release/packages/fv_all/parse_kps.pl

```diff
@@ -14,5 +14,10 @@
 my $version = $1 if($kps =~ /\<Keyboard\>.+<Version\>(.*?)\<\/Version\>/s);
 my $bcp47 = $1 if($kps =~ /\<Keyboard\>.+<Language\s+ID="(.+?)"\>/s);
 my $langname = $1 if($kps =~ /\<Keyboard\>.+<Language.+?\>(.+?)\<\/Language\>/s);
+# Font paths are optional
+my $oskFont = 'none';
+$oskFont = $1 if($kps =~ /\<Keyboard\>.+<OSKFont\>(.*?)\<\/OSKFont\>/s);
+my $displayFont = ' ';
```

Comment:
Should this be `none`?
```suggestion
my $displayFont = 'none';
```